diff --git a/node_modules/axios/index.d.ts b/node_modules/axios/index.d.ts
index e194461..bd40c68 100644
--- a/node_modules/axios/index.d.ts
+++ b/node_modules/axios/index.d.ts
@@ -353,7 +353,7 @@ export interface AxiosRequestConfig<D = any> {
 export type RawAxiosRequestConfig<D = any> = AxiosRequestConfig<D>;
 
 export interface InternalAxiosRequestConfig<D = any> extends AxiosRequestConfig<D> {
-  headers: AxiosRequestHeaders;
+  headers?: AxiosRequestHeaders;
 }
 
 export interface HeadersDefaults {
